cpp/template/cmake/thirdparty/get_raft.cmake

```diff
@@ -33,12 +33,6 @@ function(find_and_configure_raft)
     #-----------------------------------------------------
     # Invoke CPM find_package()
     #-----------------------------------------------------
-    # Since the RAFT_NVTX option is used by targets generated by
-    # find_package(RAFT_NVTX) and when building from source we want to
-    # make `RAFT_NVTX` a  cache variable so we get consistent
-    # behavior
-    #
-    set(RAFT_NVTX ${PKG_ENABLE_NVTX} CACHE BOOL "Enable raft nvtx logging" FORCE)
```

Comment:
I think we still want these changes around `nvtx` so that we keep consistent behavior. Can we unrevert this part

Reply:
Good call, done.